data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -221,10 +226,46 @@ def write_low_to_file():
             )
             logger.info("Completed writing usa-low")
 
+        def write_esri_shapefile():
+            logger.info("Producing ESRI shapefiles")
+            # Note that esri shapefiles can't have super
+            # long column names, so we borrow from the
+            # tile names. We reconstruct the dictionary in case:
+            # 1. the set of columns changes, and
+            # 2. the length of column names changes
+            # Because these are shortened names, we should
+            # also print a key of these names.
+            codebook = {}
+            renaming_map = {}
+
+            # allows us to quickly rename / describe columns
+            reversed_tiles = {
+                short: long
+                for long, short in constants.TILES_SCORE_COLUMNS.items()
+            }
+            for column in self.geojson_score_usa_high.columns:
+                # take first 10 characters, max due to ESRI constraints
+                new_col = column[:10]
+                codebook[new_col] = reversed_tiles.get(column, column)
+                if new_col != column:
+                    renaming_map[column] = new_col
+            pd.Series(codebook).reset_index().rename(
+                # kept as strings because no downstream impacts
+                columns={0: "column", "index": "meaning"}
+            ).to_csv(self.SCORE_SHP_CODE_CSV, index=False)
+            self.geojson_score_usa_high.rename(columns=renaming_map).to_file(
```

Comment:
this renames the columns that we need to rename -- it's just a few. It's kinda ugly but I do not want to deep copy the df and I also do not want to modify it in any way because of the parallelization.